packages/components/src/slot-fill/bubbles-virtually/slot-fill-provider.tsx

```diff
@@ -70,7 +70,7 @@ function createSlotRegistry(): SlotFillBubblesVirtuallyContext {
 		const slotFills = fills.get( name );
 		if ( slotFills ) {
 			// Force update fills.
-			slotFills.forEach( ( fill ) => fill.current.rerender() );
+			slotFills.forEach( ( fill ) => fill.rerender() );
```

Comment:
This is a benefit of registering the `Rerenderable` value directly: avoids the `.current` indirection when calling.